### DISCUSSION TITLE
how do i do the thing i downloaded it for

### DISCUSSION THREAD

**Discussion by fnaf19** [2fast-team/2fast#149] [General]
i cant find the option to generate codes pls help

**Comment by jp-weber** [comment]
What steps did you follow? 

1. you need a service that supports TOTP authentication for 2fa.
2. When you set this up for this service, a QR code is displayed and a field for the validation of the code is displayed
3. Now the 2fast app is used, here a new account must be clicked and the scanning with QR code is selected.
4. after clicking, you now switch to where the QR code is displayed and after a few seconds (5) the screen becomes a little darker and you can draw a rectangle around the QR code with the mouse.
5. if you now switch back to the 2fast app, the account is pre-filled with all information
6. Click create
7. copy the generated code of the new account
8. Enter the generated code in the validation field of the service of 2.
9. finish

**Reply by fnaf19** [reply]
i figured it out but it was very dumb and annoying to do, i had to enable a settign that made me do a dumb puzzle and then i had to copy a duck image, but it showed the generated code after, weirdest verification ive ever seen tho
